lib/AppInfo/Application.php

```diff
@@ -88,4 +93,18 @@ protected function registerNotificationNotifier() {
 			];
 		});
 	}
+
+	public function registerNavigation() {
+		$urlGenerator = $this->getContainer()->getServer()->getURLGenerator();
+		$l = $this->getContainer()->getServer()->getL10N('firstrunwizard');
+
+		$this->getContainer()->getServer()->getNavigationManager()->add([
```

Comment:
Please register via info.xml instead

Reply:
@nickvergessen where is this documented?

Reply:
https://nextcloudappstore.readthedocs.io/en/latest/developer.html#info-xml (search for navigation
Or https://docs.nextcloud.com/server/13/developer_manual/app/init.html?highlight=navigation

Reply:
@nickvergessen thanks :)

Reply:
Sure, I'll prepare a pull request for that.